examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -0,0 +1,541 @@
+
+  # Step 1.1
+  # Create a budget, which can be shared by multiple campaigns.
+  campaign_budget_operation = client.get_type("CampaignBudgetOperation")
+  campaign_budget = campaign_budget_operation.create
+  campaign_budget.name = f"Campaign budget {uuid.uuid4()}"
+  campaign_budget.delivery_method = (
+      client.enums.BudgetDeliveryMethodEnum.STANDARD
+  )
+  campaign_budget.amount_micros = 500000
+
+  # Add budget.
+  try:
+      campaign_budget_response = (
+          campaign_budget_service.mutate_campaign_budgets(
+              customer_id=customer_id, operations=[campaign_budget_operation]
+          )
+      )
+  except GoogleAdsException as ex:
+      handle_googleads_exception(ex)
+
+  # Step 1.2 - Create campaign.
+  campaign_operation = client.get_type("CampaignOperation")
+  campaign = campaign_operation.create
+  campaign.name = f"Testing RSA via API {uuid.uuid4()}"
+  campaign.advertising_channel_type = (
+      client.enums.AdvertisingChannelTypeEnum.SEARCH
+  )
+
+  # Recommendation: Set the campaign to PAUSED when creating it to prevent
+  # the ads from immediately serving. Set to ENABLED once you've added
+  # targeting and the ads are ready to serve.
+  campaign.status = client.enums.CampaignStatusEnum.PAUSED
+
+  # Set the bidding strategy and budget.
+  # The bidding strategy for Maximize Clicks is TargetSpend.
+  # The target_spend_micros is deprecated so don't put any value.
+  # See other bidding strategies you can select in the link below.
+  # https://developers.google.com/google-ads/api/reference/rpc/v11/Campaign#campaign_bidding_strategy
+  campaign.target_spend.target_spend_micros = 0
+  campaign.campaign_budget = campaign_budget_response.results[0].resource_name
+
+  # Set the campaign network options.
+  campaign.network_settings.target_google_search = True
+  campaign.network_settings.target_search_network = True
+  campaign.network_settings.target_partner_search_network = False
+  # Enable Display Expansion on Search campaigns. For more details see:
+  # https://support.google.com/google-ads/answer/7193800
+  campaign.network_settings.target_content_network = True
+
+  # # Optional: Set the start date.
+  # start_time = datetime.date.today() + datetime.timedelta(days=1)
+  # campaign.start_date = datetime.date.strftime(start_time, _DATE_FORMAT)
+
+  # # Optional: Set the end date.
+  # end_time = start_time + datetime.timedelta(weeks=4)
+  # campaign.end_date = datetime.date.strftime(end_time, _DATE_FORMAT)
+
+  # Add the campaign.
+  try:
+      campaign_response = campaign_service.mutate_campaigns(
+          customer_id=customer_id, operations=[campaign_operation]
+      )
+      print(f"Created campaign {campaign_response.results[0].resource_name}.")
+      campaign_id = campaign_response.results[0].resource_name.split("/")[-1]
+  except GoogleAdsException as ex:
+      handle_googleads_exception(ex)
+  # [END Step 1 -- Create campaign and budget]
```

Comment:
As mentioned earlier, you can remove the comment formatted that looks like:

* `[START xyz]`
* `[END xyz]`

Just leave the `xyz` part.